generate-assets/src/lib.rs

```diff
@@ -220,7 +225,10 @@ fn get_extra_metadata(
 ) -> anyhow::Result<()> {
     println!("Getting extra metadata for {}", asset.name);
 
-    let url = url::Url::parse(&asset.link)?;
+    let url = match &asset.crate_name {
+        Some(crate_name) => Url::parse(&format!("https://crates.io/crates/{crate_name}")),
+        None => Url::parse(&asset.link),
+    }?;
```

Comment:
I'd be a bit confused if I submitted a link to my assets entry only to find that it isn't actually used. Maybe it should be made explicit in https://github.com/bevyengine/bevy-assets/pull/459 that the link is only a fallback for when the crate name is missing.

Reply:
This is only the case when determining the latest supported Bevy version. The `link` field will still be exclusively used as the place you're directed to when you click on the asset on the website.